study-builder/studies/brain/sendgrid_emails.conf

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Quick count of PEPPER_DEV_BRAIN_* templates in sendgrid shows 18 templates where as here we seem to have only 16 templates !!

Reply:
A few of those are old or not being used so they're not added to this config.

Reply:
This config should match up to all the keys we have in `vars.conf` for brain.

Reply:
ok. May be we should delete the two unused old templates.